index.html

```diff
@@ -34,7 +34,7 @@
         -o-transform: var(--scale);
         transform: var(--scale);
       }
-      h1 { font-size: var(--step-5); }
+      h1 { font-size: var(--step-4); margin: 0.3em; }
```

Comment:
Hmm maybe top margin should suffice (there's nothing in any other direction at the moment):

```suggestion
      h1 { font-size: var(--step-4); margin-block-start: 0.3em; }
```

but actually, see below, where `#hero` elements are laid out with gaps:

```css
#hero { display: flex; flex-direction: column; gap: 4vh; align-items: center; margin-block: 4vh; }
```

Increase the gap to 5vh maybe (that essentially works like applying margin between elements for layout, only that you don't need to track whether something's last in line)